internal/plugin.go

```diff
@@ -424,127 +520,176 @@ func (p *Plugin) createPolicy(policyConf *types.PolicyConfig) error {
 	return nil
 }
 
-// getPlrFromPath finds the placement rule manifest in the input manifest file. It will return
-// the name of the placement rule, the unmarshaled placement rule manifest, and an error. An error
-// is returned if the placement rule manifest cannot be found or is invalid.
-func (p *Plugin) getPlrFromPath(plrPath string) (string, map[string]interface{}, error) {
-	manifests, err := unmarshalManifestFile(plrPath)
+// getPlcFromPath finds the placement manifest in the input manifest file. The isPlR boolean
+// signals whether the path should point to a PlacementRule kind (as opposed to a Placement kind).
+// It will return the name of the placement, the unmarshaled placement manifest, and an
+// error. An error is returned if the placement manifest cannot be found or is invalid.
+func (p *Plugin) getPlcFromPath(plcPath string, isPlR bool) (string, map[string]interface{}, error) {
```

Comment:
Is the `isPlR` argument needed since after running the validation (i.e. `assertValidConfig`) you know that if `p.usingPlR` is set that only placement rules are expected and vice-versa?

Reply:
Yes because a user could use `placementRulePath` but specify a Placement kind there. `assertValidConfig` wouldn't be able to verify that. But that's a good point. The use of `placementRulePath` vs `placementPath` could be handled before entering the manifest retrieval. I'll update the code.

Reply:
I thought about it more, and I see your point now. Yes, I'll scrap `isPlR`.